e2e/rbd.go

```diff
@@ -433,6 +456,15 @@ var _ = Describe("RBD", func() {
 						go func(w *sync.WaitGroup, n int, p v1.PersistentVolumeClaim, a v1.Pod) {
 							name := fmt.Sprintf("%s%d", f.UniqueName, n)
 							wgErrs[n] = createPVCAndApp(name, f, &p, &a, deployTimeout)
+							filePath := appClone.Spec.Containers[0].VolumeMounts[0].MountPath + "/test"
+							checkSumClone, wErr := calculateSHA512sum(f, &a, filePath, &opt)
+							if wErr != nil {
```

Comment:
currently, we are not failing a test case inside a goroutine, please add it to wgErrs[n] error slice

Reply:
okay